app/src/main/java/com/example/harvestfresh/fragments/CartFragment.java

```diff
@@ -0,0 +1,84 @@
+package com.example.harvestfresh.fragments;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.example.harvestfresh.Cart;
+import com.example.harvestfresh.CartAdapter;
+import com.example.harvestfresh.R;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CartFragment extends Fragment {
+
+    private static final String TAG = "CartFragment";
+    private static final String PRODUCT_DISPLAY = "Cart ";
+
+    private RecyclerView rvCart;
+    private CartAdapter fragmentAdapter;
+    private List<Cart> allCarts;
+
+    public CartFragment() {
+    }
+
+    public static CartFragment newInstance() {
+        CartFragment fragment = new CartFragment();
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.fragment_cart, container, false);
+    }
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        rvCart = view.findViewById(R.id.rvCart);
+        rvCart.setLayoutManager(new LinearLayoutManager(getContext()));
+
+        allCarts = new ArrayList<>();
+        fragmentAdapter = new CartAdapter(getContext(), allCarts);
+        rvCart.setAdapter(fragmentAdapter);
+
+        queryCart();
+    }
+
+    private void queryCart() {
+        ParseQuery<Cart> query = ParseQuery.getQuery(Cart.class);
+        query.setLimit(20);
```

Comment:
Extract const: This makes it easier to understand where the value is coming from and change it in the future, especially if there get to be additional references.